src/Page/Form.purs

```diff
@@ -204,6 +211,27 @@ component =
         , HP.id_ field.key
         , HE.onChecked (HE.input $ UpdateKey field.key <<< UserInput <<< Lynx.Data.Expr.Boolean)
         ]
+    TypeaheadSingle typeahead ->
+      HH.slot'
+        cp3
+        field.key
+        Typeahead.single
+        ( Typeahead.syncSingle
+          { itemToObject: \item -> fold do
+            { name: name', value: value' } <- toPair item
+            name <- toString name'
+            value <- toString value'
+            pure (Foreign.Object.fromHomogeneous { name, value })
+          , renderFuzzy: \(Fuzzy { original }) -> HH.text $ fold do
```

Comment:
We'll want to include match highlighting in our results. See Ocelot example:
https://github.com/citizennet/purescript-ocelot/blob/b71ad0e41745a9146119914dac8a21900b511a6e/ui-guide/Components/Typeaheads.purs#L159